download_task.rb

```diff
@@ -0,0 +1,206 @@
+require 'fileutils'
```

Comment:
I am not able to digest this ruby file wandering in root path. Can we move to some folder? 

What about `features/support` folder? Since it being used for that purpose.